account_analytic_required/account_invoice.py

```diff
@@ -0,0 +1,37 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#
+#    Copyright (c) 2015 Noviat nv/sa (www.noviat.com).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class account_invoice_line(models.Model):
+    _inherit = 'account.invoice.line'
+
+    @api.model
+    def _get_policies(self):
+        return self.pool["account.account.type"]._get_policies()
+
+    analytic_policy = fields.Selection(
+        '_get_policies', string='Policy for analytic account',
+        related='account_id.user_type.analytic_policy')
```

Comment:
@sbidoul 
I tested without compute method and it works.
I have updated the PR. 
Thanks for the valuable suggestion.